doc/API-Entities.md

```diff
@@ -0,0 +1,1458 @@
+# Friendica API entities
+
+* [Home](help)
+  * [Using the APIs](help/api)
+
+
+## Activities
+
+<table class="table table-condensed table-striped table-bordered">
```

Comment:
Why didn't up you use the markdown specific way of creating tables? https://help.github.com/en/github/writing-on-github/organizing-information-with-tables I'd prefer it over the plain HTML :)

Reply:
No way of styling the generated tables.